docs/swupd.1

```diff
@@ -400,8 +400,8 @@ Examples:
 Ignores \fB/var\fP or \fB/etc/machine\-id\fP, regardless of
 whether they are directories or something else. In the
 usual case that \fB/var\fP is a directory, also everything
-inside it is ignored because directories get skipped
-during scanning the directory tree.
+inside it is ignored because the directory gets skipped
+while scanning the directory tree.
```

Comment:
Unrelated change? I would separate this into a separate commit if it's a valid change.

Reply:
sure.

Reply:
Moved to own commit.